### PR TITLE
Added Attribute  Selector question in css-quiz.ts

### PR DESCRIPTION
**Pull request by Ankit-Aslekar**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/css-quiz.ts [hunk 1]
```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
     Explanation:
       "We can use the :last-of-type selector to target an element of a specific type within a container.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type/"
+  },
+  {
+    Question:
+      "what is syntax of Attribute Selector in CSS?",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1203]:
```suggestion
      "Which of the following is the correct syntax for an attribute selector in CSS?",
```

File: src/data/css-quiz.ts [hunk 2]
```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
...
+    Answer: "container[attribute="value"]",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1204]:
```suggestion
    Answer: "selector[attribute='value']",
```

File: src/data/css-quiz.ts [hunk 3]
```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
...
+    Distractor1: "container[value]",
+    Distractor2: "attribute[value]",
+    Distractor3: "value[attribute]",
+    Explanation:
+      "It selects elements with an attribute with specefic value .",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1209]:
```suggestion
      "The correct syntax for an attribute selector in CSS is selector[attribute='value'].",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Ankit-Aslekar** [comment]
@jdwilkin4 thanks for reviewing, I too liked the changes and updated PR, please have look again

File: src/data/css-quiz.ts (hunk 2 above)
Comment (jdwilkin4) [reply src/data/css-quiz.ts]: @Ankit-Aslekar 

You have to use single quotes like suggested above to fix the linting errors

File: src/data/css-quiz.ts [hunk 4]
```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
     Explanation:
       "We can use the :last-of-type selector to target an element of a specific type within a container.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type/"
+  },
+  {
+    Question:
+      "Which of the following is the correct syntax for an attribute selector in CSS?",
+    Answer: "selector[attribute="value"]",
```
Comment (jdwilkin4) [inline src/data/css-quiz.ts:1204]:
```suggestion
    Answer: "selector[attribute='value']",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Comment by jdwilkin4** [comment]
I went ahead and resolved the syntax linting error

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**